app/Notifications/EventSuggestionProcessed.php

```diff
@@ -11,12 +12,14 @@ class EventSuggestionProcessed extends Notification implements BaseNotification
 {
     use Queueable;
 
-    private EventSuggestion $eventSuggestion;
-    private ?Event          $event;
+    private EventSuggestion      $eventSuggestion;
+    private ?Event               $event;
+    private ?EventRejectionReason $reason;
 
-    public function __construct(EventSuggestion $eventSuggestion, ?Event $event) {
+    public function __construct(EventSuggestion $eventSuggestion, ?Event $event, ?EventRejectionReason $reason = null) {
```

Comment:
```suggestion
    public function __construct(EventSuggestion $eventSuggestion, ?Event $event, EventRejectionReason $reason = null) {
```

don't need to be nullable if the default is null.